e2e/framework/utils/k8s_helper.go

```diff
@@ -304,3 +286,45 @@ func (k8sh *K8sHelper) GetPodHostId(podNamePattern string, namespace string) (st
 		return serr, fmt.Errorf("Error Getting Monitor IP")
 	}
 }
+
+func (k8sh *K8sHelper) IsStorageClassPresent(name string) (bool, error) {
+	cmdArgs := []string{"get", "storageclass", "-o", "jsonpath='{.items[*].metadata.name}'"}
+	sout, serr, _ := ExecuteCmd("kubectl", cmdArgs)
```

Comment:
ok, let's not delay that work. the more helpers we add on top of kubectl, the more painful to convert later.